### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by jburel**
delete files not longer needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,12 +14,11 @@
...
 
-    + build: This directory contains the tools to compile, run, test, and
-          deliver the application.
-    + config: Various configuration files required by the application to run.
-    + launch: Its sub-dirs contain further resources to build platform-specific distributions.
-    + src: Contains the application source files.
-    + test: The test code.
+    + build: This directory contains the tools to compile, run, test and
+             deliver the application.
+    + config:
```
Comment (mtbc) [inline README.md]: Odd it's empty.

**Comment by mtbc** [comment]
Has importer-cli generally gone? Ah well, it was nice while we had it.

**Comment by jburel** [comment]
The importer scripts will still be generated.

**Comment by jburel** [comment]
Ok to merge @mtbc?

**Comment by mtbc** [comment]
I don't see any other problems with it! :+1:

**Pull request merged**